src/commonmark/plugins/preview.ts

```diff
@@ -31,9 +31,7 @@ class PreviewView implements PluginView {
             previewOptions?.renderer ||
             createDefaultMarkdownItInstance({
                 ...parserFeatures,
-                // TODO until we handle proper html sanitizing in the renderer,
-                // we need to disable html entirely...
-                html: false,
+                html: true,
```

Comment:
We _cannot_ turn this on until the TODO comment (that was deleted here) is addressed. This is a major security vulnerability (self XSS) that I'm not willing to ship. If consumers want to override it, they can.